src/dirhash/_version.py

```diff
@@ -8,6 +8,8 @@
 # Generated by versioneer-0.29
 # https://github.com/python-versioneer/python-versioneer
 
+# ruff: noqa
```

Comment:
why is this noqa necessary ?

Reply:
This whole file is autogenerated by `versioneer`. Should never be edited manually, so better to not touch formatting either.

Reply:
-> would be better to exclude outside (in pre-commit-config), I'll fix that :)

Reply:
So the comment is a quick fix to make ruff ignore the entire file.